lib/NfcoreTemplate.groovy

```diff
@@ -32,6 +32,21 @@ class NfcoreTemplate {
         }
     }
 
+    //
+    //  Warn if using custom configs to provide pipeline parameters
+    //
+    public static void warnParamsProvidedInConfig(workflow, log) {
+        if (workflow.configFiles.size() > 1) {
+            log.warn "=============================================================================\n" +
```

Comment:
NO `====`! You will mess with git merge conflicts!!!!!!! :grimacing: :grimacing: :grimacing: :grimacing: :grimacing: :grimacing:

Reply:
Ohhhh... I hav these all over rnaseq...which symbols work best here?